core/controllers/topics_and_skills_dashboard_test.py

```diff
@@ -58,7 +58,7 @@ def test_get(self):
             response = self.testapp.get(
                 '%s' % feconf.TOPICS_AND_SKILLS_DASHBOARD_DATA_URL,
                 expect_errors=True)
-            self.assertEqual(response.status_int, 401)
+            self.assertEqual(response.status_int, 200)
```

Comment:
Why is an "expect_errors=True" response giving a status code of 200?

More generally, why did this test change?

Reply:
When a user tries to create a question, we need to list out the topics. But that user does not have permissions to get the topics list data. I had asked @aks681 what I should do, and he suggested to change the permissions of the get handler for this controller. I had made it open access, as any user should be able to create a question suggestion and we need to retrieve topic list for creation. 

As for the first question, I missed out on removing expect_errors=True. Will do!

Reply:
This should not change. See above.

Reply:
Done